### DISCUSSION TITLE
Documenting SEA implementations outside the context of Node.js?

### DISCUSSION THREAD

**Discussion by jviotti** [nodejs/single-executable#5] [Ideas]
Do you think it would make sense to also document SEA implementations that are not directly applicable to Node.js? For example, one could argue that [AppImage](https://appimage.github.io) is a generic language-agnostic SEA implementation for GNU/Linux.

While its on one side interesting from a research-completeness point of view and might provide some inspiration, we might want to limit the research to Node.js just to not increase the scope too much.

Any thoughts?

**Comment by RaisinTen** [comment]
I think we should consider all kinds of SEA solutions on a case-by-case basis depending on the approach each of these solutions take, advantages, limitations, etc. Mentioning such info would be helpful in judging if it makes sense to be documented here.
